shaptools/hana.py

```diff
@@ -448,3 +449,106 @@ def get_sr_status(self):
         # (see SAPHana RA)
         status["status"] = SR_STATUS.get(result.returncode, SR_STATUS[12])
         return status
+
+    def set_ini_parameter(
+            self, ini_parameter_values, database, file_name, layer,
+            layer_name=None, reconfig=False,
```

Comment:
I would prefer to have `reconfig` and `layer_name` in `kwargs`. We have too many parameters otherwise,